### DISCUSSION TITLE
Petite erreur dans la doc ?

### DISCUSSION THREAD

**Discussion by Fr3d3ric-3s** [picsou83/compta.libremen.com#4] [General]
Bonjour,

tout d'abord merci pour votre travail ;)

Il y a une petite coquille dans la doc il y est indiqué `rep_bdd:/var/lib/postgresql/` dans la commande `docker run` mais pour que le conteneur démarre sans erreur il faut mettre `rep_bdd:/var/lib/postgresql/data` sinon on a une jolie erreur `"Starting PostgreSQL 13 database server: mainError: /var/lib/postgresql/13/main is not accessible or does not exist ... "` au démarrage.

Sinon comment utiliser le adminer livré dans le dossier app  ?

Bonnes fêtes ^^

**Comment by picsou83** [comment]
Bonjour,

C'est étrange, car pour ma part, je n'en ai pas besoin :

Sur mon environnement Rocky Linux release 8.7, je lance l'image avec la commande suivante :

`sudo docker run -i --name comptalibremen -t -v rep_app:/var/www/html/Compta/ -v rep_bdd:/var/lib/postgresql/ --restart always -d picsou83/compta-libremen-com:latest`

L'image déployée à des fins de tests depuis la plateforme cloud Oracle et hébergée sur une machine CentOS 8 est également démarrée avec la commande suivante :

`sudo docker run -i --name comptalibremen -t -v rep_app:/var/www/html/Compta/ -v rep_bdd:/var/lib/postgresql/ -p 80:80 -d --restart "unless-stopped" picsou83/compta-libremen-com:latest`

Pourriez-vous partager des informations sur votre environnement, notamment la machine utilisée (Linux, OS, etc.) ?

Quant à Adminer, je l'ai ajouté à des fins de débogage. Vous pouvez y accéder via les liens suivants :

http://xxx.xxx.xxx.xxx/Compta/adminer.php

Pour accéder directement à la base de données :
http://xxx.xxx.xxx.xxx/Compta/adminer.php?pgsql=localhost&username=compta&db=comptalibre&ns=public&password

N'oubliez pas de remplacer xxx.xxx.xxx.xxx par l'adresse IP correcte de votre machine. 

Si vous avez des questions supplémentaires ou des ajustements spécifiques à faire, n'hésitez pas à me le faire savoir !

**Comment by Fr3d3ric-3s** [comment]
Bonsoir,

Je suis sur un NAS Synology avec Portainer pour gérer les conteneurs. Et pour les volumes je dois mettre cela pour que ca marche

![image](https://github.com/picsou83/compta.libremen.com/assets/33122997/f091111a-e984-4b6e-b84c-9630f1e85f9a)

Pour la version c'est un Linux DSM 7.2.1-69057 Update 3 en linux 4.4302+

```
~$ uname -a
Linux NAS-2024 4.4.302+ #69057 SMP Mon Nov 13 14:19:11 CST 2023 x86_64 GNU/Linux synology_r1000_723+

$ sudo docker version
Client:
 Version:           20.10.23
 API version:       1.41
 Go version:        go1.19.1
 Git commit:        876964a
 Built:             Fri Sep  8 08:19:48 2023
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.23
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.19.1
  Git commit:       5fcb1cf
  Built:            Fri Sep  8 08:18:18 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.6.15
  GitCommit:        b23a389d8c181697302d163356e97dec04eb8d88
 runc:
  Version:          v1.1.4
  GitCommit:        5af893d
 docker-init:
  Version:          0.19.0
  GitCommit:        ed96d00

```

Merci pour adminer !

Pour les ajustement peut-être pouvoir changer le couple compte/mot de passe postgreSQL via des variables d'environnement au 1er démarrage du conteneur et éventuellement pouvoir avoir la base sur un conteneur postgre à part, avec des variables de ce type : 

![image](https://github.com/picsou83/compta.libremen.com/assets/33122997/f7749659-d3fe-490c-9a6d-b795fe1ee7cd)

Merci !
